### PR TITLE
Legacy mastheads

### PR DESCRIPTION
**Pull request by Sean-BREI**
Note: One Format was not updated to the most recent changes during my last Pull Request (1-Column Dropdown Feature) -- primary_column_helpers.velocity; that Format with those updates have been included in this Pull Request.

### CODE REVIEW

**Comment by mandythomas** [comment]
@Sean-BREI  since you added in a file that is part of your prior Trello card and pull request, do we need to review the published pages again that use this format or did you already put it in Cascade and the website yesterday? Also, do you know why it lives in /helper subfolder? Is it for testing and not used on the site?

**Comment by Sean-BREI** [comment]
@mandythomas No worries about reviewing live content, or published pages. That Format was updated for both the Dev and Production CMS during integration. I just forgot to update the Format again in the repo after additional changes were requested prior. That's my mistake; but I wanted to get that change included somehow.

As for its location and where it's stored within the CMS... that I cannot answer. However, it's used to render content for the 1-Column layout such as a heading, description, category and call-to-action button along with the newly added dropdown feature.

File: app/assets/stylesheets/themes/law.scss [hunk 1]
```diff
@@ -126,44 +126,68 @@
...
+  }
+
+  .styleTwo h2 {
+    color: $white;
+  }
+
+  .styleTwo h3 {
+    color: $white;
+  }
+
```
Comment (cbryant24) [inline app/assets/stylesheets/themes/law.scss]: Can we simplify this so all these selectors are combined, or is there a specific reason they're all separated

`.styleTwo p, .styleTwo li,` 
`.styleTwo li,  .styleTwo div,`
`.styleTwo h2, .styleTwo h3 {`
  `color: $white;`
`}`

File: .cascade-code/Chapman.edu/_cascade/formats/level/Masthead.vtl [hunk 2]
```diff
@@ -3,322 +3,178 @@
...
+	</div>
+	<div class="miniRotatorNav">
+		<a class="prev" title="Move back to the previous slide"></a>
+		<a class="next" title="Move forward to the next slide"></a>
+	</div>
 
 #else
-    <!-- mastheadType: base case (no match, no masthead) -->
-
+	<!-- mastheadType: base case (no match, no masthead) -->
```
Comment (cbryant24) [inline .cascade-code/Chapman.edu/_cascade/formats/level/Masthead.vtl]: The spacing is off on this file it looks like it was 2 spaces and is now 4 spaces can you change this

File: app/assets/stylesheets/themes/law.scss (hunk 1 above)
Comment (Sean-BREI) [reply app/assets/stylesheets/themes/law.scss]: No reason that I can identify. I've reformatted this style block as so...

  .styleTwo {

    p,
    li,
    div,
    h2,
    h3 {
      color: $white;
    }
  }

File: .cascade-code/Chapman.edu/_cascade/formats/level/Masthead.vtl (hunk 2 above)
Comment (Sean-BREI) [reply .cascade-code/Chapman.edu/_cascade/formats/level/Masthead.vtl]: Updated. My Text Editor must have converted two spaces to four. Let me know if this file still isn't displaying correctly for you.

**Pull request merged**